python/pyhail/dataset.py

```diff
@@ -830,13 +1081,43 @@ def hardcalls(self):
     def ibd(self, output, maf=None, unbounded=False, min=None, max=None):
         """Compute matrix of identity-by-descent estimations.
 
+        **Examples**
+
```

Comment:
I'd make the first example the simplest possible, which is no MAF pre-computed correct and default arguments.